src/PackV2.ts

```diff
@@ -110,14 +108,31 @@ export const run = Effect.gen(function*(_) {
     return out
   })
 
+  const writeDtsFor = (module: string) =>
+    Effect.all([
+      fs.writeFileString(
+        `dist/dist/cjs/${module}.d.ts`,
+        `export * from "../dts/${module}"`,
```

Comment:
u could very well just do this to be more explicit:
```suggestion
        `export * from "../dts/${module}.js"`,
```

the `.js` extension isn't forbidden in CJS-oriented `.d.ts` files